src/javascript/app/ng-wig/ngWigDirective.js

```diff
@@ -24,6 +24,13 @@ angular.module('ngWig').directive('ngWig', function () {
             }
             scope.$emit('execCommand', {command: command, options: options});
           };
+
+          scope.styles = [
+            {name: 'Normal text', value: 'p'},
+            {name: 'Header 1', value: 'h1'},
+            {name: 'Header 2', value: 'h2'},
+            {name: 'Header 3', value: 'h3'}
+          ];
         }
```

Comment:
What is "no style at all"?
All text content should semantically be kept in corresponding nodes. A block of text is either a paragraph, a list, a header or something else, but not just "something". Or will someone want to get "not a header and not a normal text, but something else"?)